source/codegen/metadata/niscope/enums.py

```diff
@@ -1427,20 +1257,6 @@
             }
         ]
     },
-    'NiScopeInt64AttributeValues': {
```

Comment:
Why is `NiScopeInt64AttributeValues` missing?

Reply:
So there was only one Int64 attribute referencing an enum previously which was `P2P_NOTIFY_MESSAGE_PUSH_ADDR`. There was a conversation about it in Dane's original Draft PR and Harsha agreed that that attribute shouldn't have been marked with that enum as it didn't make sense as a possible input for that attribute.

So that 'enum' tag was removed from that attribute which results in that enum's values being removed from this `Int64AttributeValues`. Since it was the only enum here the whole Int64AttributeValues dissappears because it'd be empty and it wouldn't be referenced by any Request/Response messages anymore.